### PR TITLE
Formatting changes

### PR DESCRIPTION
**Pull request by Masterzach32**
This is something we should use a PR for, as it changes how the code looks, and could change functionality. I'll use the next few days to go over each change individually

### CODE REVIEW

File: src/main/java/org/team401/lib/ADXRS450_Gyro.java [hunk 1]
```diff
@@ -28,19 +28,19 @@
...
+
+	private static final int kRateRegister = 0x00,
+		kTemRegister = 0x02,
+		kLoCSTRegister = 0x04,
+		kHiCSTRegister = 0x06,
+		kQuadRegister = 0x08,
+		kFaultRegister = 0x0A,
+		kPIDRegister = 0x0C,
+		kSNHighRegister = 0x0E,
+		kSNLowRegister = 0x10;
```
Comment (Masterzach32) [inline src/main/java/org/team401/lib/ADXRS450_Gyro.java]: Im fine with these changes

File: src/main/java/org/team401/lib/Lidar.java [hunk 2]
```diff
@@ -33,8 +33,8 @@
 		private Hardware hardware;
 		private Unit unit;
 		private AtomicReference<Data> latestData;
-		private byte[] buffer = {0,0};
-		private byte[] writeBuffer = new byte[1];
+		private byte[] buffer = {0,0},
+			writeBuffer = new byte[1];
```
Comment (Masterzach32) [inline src/main/java/org/team401/lib/Lidar.java]: I would be careful with this, ask @camearle20 about changing his lidar code

File: src/main/java/org/team401/lib/SynchronousPID.java [hunk 3]
```diff
@@ -176,15 +175,25 @@ public void setOutputRange(double minimumOutput, double maximumOutput) {
...
+		/*if (maximumInput > minimumInput)
 			if (setpoint > maximumInput)
 				this.setpoint = maximumInput;
 			else if (setpoint < minimumInput)
 				this.setpoint = minimumInput;
 			else
 				this.setpoint = setpoint;
 		else
-			this.setpoint = setpoint;
+			this.setpoint = setpoint;*/
```
Comment (Masterzach32) [inline src/main/java/org/team401/lib/SynchronousPID.java]: I don't think you should do this here, it is much harder to read.

File: src/main/java/org/team401/robot/Robot.java [hunk 4]
```diff
@@ -147,7 +149,7 @@ public void robotInit() {
 			enabledLoop.register(data);
 			disabledLoop.register(data);
 
-			System.out.println("Done! Setting cameras to stream mode...");
+			System.out.print("Done!\nSetting cameras to stream mode...");
```
Comment (Masterzach32) [inline src/main/java/org/team401/robot/Robot.java]: I actually like this change, should look cleaner.

File: src/main/java/org/team401/robot/subsystems/Turret.java [hunk 5]
```diff
@@ -9,273 +9,244 @@
...
+	}
+
+	public Loop getSubsystemLoop() {
+		return loop;
+	}
+
+	public static Turret getInstance() {
+		return instance;
+	}
 }
```
Comment (Masterzach32) [inline src/main/java/org/team401/robot/subsystems/Turret.java]: Is there anything different here?

**Review by Masterzach32 — COMMENTED** [review]
Overall just a few changes need final review. @Briangulator you need @Profious and @camearle20 to review and accept your formatting changes to their code.

File: src/main/java/org/team401/lib/Lidar.java (hunk 2 above)
Comment (camearle20) [reply src/main/java/org/team401/lib/Lidar.java]: This shouldn't be a problem, but may affect consistency around other parts of the file.  Also, I may be confusing this for C++, but will this actually give us the desired separation, or will this create a bound reference?

**Pull request closed without merging**